presto-testng-services/src/main/java/io/prestosql/testng/services/Flaky.java

```diff
@@ -28,4 +29,10 @@
 public @interface Flaky
 {
     String issue();
+
+    /**
+     * A test is retried when it fails with a stacktrace, which string representation matches given regular expression.
+     * The pattern is searched for, as if {@link Matcher#find()} was used.
+     */
+    String matching();
```

Comment:
Would `match` be better name?

Reply:
exceptionMessageRegExp? It doesn't have to be super user friendly :)